client/src/app/queries/applications.ts

```diff
@@ -71,6 +72,24 @@ export const useFetchApplicationById = (id?: number | string) => {
   };
 };
 
+export const useFetchApplicationManifestById = (id?: number | string) => {
```

Comment:
The manifest is being fetched by the application id, not the manifest id.

```suggestion
export const useFetchApplicationManifest = (applicationId?: number | string) => {
```